projects/packages/connection-ui/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "automattic/jetpack-connection-ui",
+	"description": "Jetpack Connection UI",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"automattic/jetpack-connection": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"scripts": {
+		"build-development": [
+			"Composer\\Config::disableProcessTimeout",
+			"yarn build"
+		],
+		"build-production": [
+			"Composer\\Config::disableProcessTimeout",
+			"yarn build"
+		]
+	},
+	"minimum-stability": "dev",
+	"prefer-stable": true
+}
```

Comment:
I believe we should add `extra.mirror-repo` here, as per the requirements:
https://github.com/Automattic/jetpack/blob/5c5ead66f8e7e59f93423eb7f34a54b5132daca2/docs/monorepo.md#L46